stl/inc/compare

```diff
@@ -679,7 +679,7 @@ concept _Can_fallback_eq_lt = requires(_Ty1& _Left, _Ty2& _Right) {
 // CUSTOMIZATION POINT OBJECT compare_strong_order_fallback
 namespace _Compare_strong_order_fallback {
     template <class _Ty1, class _Ty2>
-    concept _Can_strong_order = requires(_Ty1& _Left, _Ty2& _Right) {
+    concept _Can_strong_order = requires(_Ty1 & _Left, _Ty2 & _Right) {

```

Comment:
That one is really strange, No change requested though

Reply:
Yeah, I don't know why clang-format is adding spaces to these parameters, but not to nearby concepts. While it's indeed strange, I didn't think it affected readability enough to be worth suppressing clang-format.